admin/common_auth/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class AdminUser(models.Model):
```

Comment:
@Ghalko Please see my other comment on the original purpose of extending the User model.
However, I do agree. We only need the default User model, and using it guarantees no unexpectedness when it comes to django authentication or the admin backend.